docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -52,18 +52,26 @@ GBD Causes
 
 * :ref:`Diarrheal Diseases <2017_cause_diarrhea>`
 
-* :ref:`Neural Tube Defects <2017_cause_neural_tube_defects>`
-
 * :ref:`Lower Respiratory Infections <2017_cause_lower_respiratory_infections>`
 
-* :ref:`Neonatal Encephalopathy <2017_cause_neonatal_encephalopathy>`
-
-* :ref:`Neonatal Sepsis <2017_cause_neonatal_sepsis>`
-
-* :ref:`Neonatal Jaundice <2017_cause_neonatal_jaundice>`
+* :ref:`Neural Tube Defects <2017_cause_neural_tube_defects>`
 
-* :ref:`Other Neonatal Disorders <2017_cause_neonatal_other>`
-  (Probably omit this cause, as we don't seem to have enough data to model it)
+.. note::
+
+  Although the sub-causes of  :ref:`Neonatal Disorders
```

Comment:
Maybe move this to an assumptions and limitations section?

Reply:
I'm going to merge this version into master and create a new PR to move this note into a new section and expand on it.